opm/autodiff/SimulatorFullyImplicitBlackoilOutputEbos.hpp

```diff
@@ -743,5 +745,20 @@ namespace Opm
 
         writeTimeStepWithCellProperties(timer, localState, localWellState, cellData, substep);
     }
+
+
+    bool BlackoilOutputWriterEbos::hasFIPSummaryKeywords() const
+    {
+      const SummaryConfig& summaryConfig = eclipseState_.getSummaryConfig();
+      return detail::hasFRBKeyword(summaryConfig, "WIP")  ||
```

Comment:
it is called in the opm-core part. anyways, i have rewritten to use the method from parser but it does not seem to do the job. trying to get a debug session going but as i said, my time is not my own ;)